### DISCUSSION TITLE
How to place a row or control on the lower edge of the device?

### DISCUSSION THREAD

**Discussion by nascin** [flet-dev/flet#183] [Q&A] [answered]
How to place a line or control on the lower edge of the device (desktop or mobile browser)? Like a bottom navigation bar.

**Comment by pamoroso** [comment]
This may be done with a `NavigationBar` control, which is on the [Flet roadmap](https://flet.dev/docs/roadmap#controls).

**Comment by FeodorFitsner** [comment, marked as answer]
Yeah, it's going to be `NavigationBar`: https://api.flutter.dev/flutter/material/NavigationBar-class.html
Subscribe to this issue please: https://github.com/flet-dev/flet/issues/193
